docs/docs/reference-glossary.md

```diff
@@ -0,0 +1,100 @@
+
+## Package Manager 
+Package managers are tools that allow you to manage dependencies in your project. npm & Yarn are two package managers commonly used in React applications
+
+## CDN
+CDN stands for Content Delivery Network. CDNs deliver cached, static content from a network of servers across the globe. 
+
+## JSX
+JSX is a syntax extension to JavaScript. It is similar to a template language. When JSX gets compiled, the result is JavaScript objects called "elements". To get a basic introduction to JSX [see the docs here](https://facebook.github.io/react/docs/introducing-jsx.html) and find a more in-depth tutorial on JSX [here](https://facebook.github.io/react/docs/jsx-in-depth.html)
+
+React DOM uses camelCase property naming convention instead of HTML attribute names.
+For example, class becomes className in JSX, and tabindex becomes tabIndex.
+
+## [Elements](https://facebook.github.io/react/docs/rendering-elements.html)
+React elements are the building blocks of React applications. One might confuse elements with a more widely known concept of "components". Elements are what components are "made of".An element describes what you want to see on the screen. React elements are immutable.
+
+```js
+const element = <h1>Hello, world</h1>;
+```
+
+## [Components](https://facebook.github.io/react/docs/components-and-props.html)
+React components are small, resuable pieces of code that return a React element to be rendered to the page. The simplest version of React component is a plain JavaScript function that returns a React element:
+
+```js
+function Welcome(props) {
+  return <h1>Hello, {props.name}</h1>;
+}
+```
+
+Components can also be ES6 classes:
+
+```js
+class Welcome extends React.Component {
+  render() {
+    return <h1>Hello, {this.props.name}</h1>;
+  }
+}
+```
+
+Components can be broken down into distinct pieces of functionality and used within other components. Components must return a single root element (wrapped in a `div` or other similar container). A good rule of thumb is that if a part of your UI is used several times (Button, Panel, Avatar), or is complex enough on its own (App, FeedStory, Comment), it is a good candidate to be a reusable component. Component names should also always start with a capital letter (`<Wrapper/>` **not** `<wrapper/>`). 
+
+
+### [`props`](https://facebook.github.io/react/docs/components-and-props.html)
+`props` are inputs to a React component. They are data passed down from a parent component to a child component. `props` are readonly -- they should not be modified in any way. All React components must act like pure functions with respect to their `props`.
+
+### `this.props.children`
+`this.props.children` is available on every component. It contains the content between the opening and closing tags of a component. For example:
```

Comment:
For function components, it would just be `props.children`, since you access it on the passed parameter and not `this`